### PR TITLE
Refactored code to automatically add new products added to 0vix.com

### PR DESCRIPTION
**Pull request by chaoticgoodpanda**
Since we're adding new strategies to 0vix.com, refactored code to automatically add these new strategies to defi-llama for APYs/yield. Passes all tests.

### CODE REVIEW

**Comment by vrtnd** [comment]
Please resolve conflicts

**Comment by llamatester** [comment]
Error while running 0vix adapter: 
        
 
 Test Suites: 1 failed, 1 total
    Tests:       1 failed, 38 passed, 39 total
    Snapshots:   0 total
    Time:        0.426 s
    Ran all test suites.
    
    Nb of pools: 7
     
    
    Sample pools:
    ┌─────────┬──────────────────────────────────────────────┬─────────┬─────────────────────────────┬───────────┬─────────────────────┬────────────────────┬────────────────────┐
    │ (index) │                     pool                     │ project │           symbol            │   chain   │         apy         │      apyBase       │       tvlUsd       │
    ├─────────┼──────────────────────────────────────────────┼─────────┼─────────────────────────────┼───────────┼─────────────────────┼────────────────────┼────────────────────┤
    │    0    │ '0xEBb865Bf286e6eA8aBf5ac97e1b56A76530F3fBe' │ '0vix'  │    '0VIX USD Coin (PoS)'    │ 'polygon' │ 0.5770237962472669  │ 2.7180863964237245 │ 410673.31831047824 │
    │    1    │ '0xb2D9646A1394bf784E376612136B3686e74A325F' │ '0vix'  │    '0VIX Wrapped Ether'     │ 'polygon' │ 0.21931859714465762 │ 1.6686030951213349 │ 236607.31296332413 │
    │    2    │ '0xE554E874c9c60E45F1Debd479389C76230ae25A8' │ '0vix'  │        '0VIX MATIC'         │ 'polygon' │  4.48909930990411   │  7.6901851603002   │ 198846.66354884324 │
    │    3    │ '0x3B9128Ddd834cE06A60B0eC31CCfB11582d8ee18' │ '0vix'  │  '0VIX (PoS) Wrapped BTC'   │ 'polygon' │  2.323659059114913  │ 5.504655759753674  │ 180974.35288782002 │
    │    4    │ '0x2175110F2936bf630a278660E9B6E4EFa358490A' │ '0vix'  │ '0VIX (PoS) Dai Stablecoin' │ 'polygon' │  6.587685323079651  │ 9.341006197397883  │ 75801.20753234699  │
    │    5    │ '0x1372c34acC14F1E8644C72Dad82E3a21C211729f' │ '0vix'  │   '0VIX (PoS) Tether USD'   │ 'polygon' │ 2.7508573846971807  │ 5.997144347470096  │ 61749.82302560896  │
    │    6    │ '0xC57E5e261d49Af3026446de3eC381172f17bB799' │ '0vix'  │         '0VIX MAI'          │ 'polygon' │ 11.001696992417088  │ 18.78473064340962  │ 25407.626343784257 │
    └─────────┴──────────────────────────────────────────────┴─────────┴─────────────────────────────┴───────────┴─────────────────────┴────────────────────┴────────────────────┘
    }

**Comment by chaoticgoodpanda** [comment]
Resolved conflicts, added url, passes tests

File: src/adaptors/0vix/index.js [hunk 1]
```diff
@@ -1,150 +1,173 @@
...
+        tvl = await getErc20Balances(strategy);
+
+
+        const newObj = {
+            pool: strategy,
+            project: "0vix",
+            symbol: OvixAPYs.tokenName,
+            chain: "polygon",
+            apy: OvixAPYs.supplyAPY,
+            apyBase: OvixAPYs.borrowAPY,
```
Comment (vrtnd) [inline src/adaptors/0vix/index.js]: Please remove borrowApy.
Borrow apy tracking will be added soon as a new field.

**Comment by chaoticgoodpanda** [comment]
Okay done, removed `borrowApy`

**Pull request merged**